llama_index/indices/postprocessor/sbert_rerank.py

```diff
@@ -0,0 +1,45 @@
+from typing import List, Optional
+from llama_index.indices.postprocessor.types import BaseNodePostprocessor
+from llama_index.indices.query.schema import QueryBundle
+from llama_index.schema import NodeWithScore
+
+
+class SentenceTransformerRerank(BaseNodePostprocessor):
+    def __init__(
+        self,
+        top_n: int = 2,
+        model: str = "cross-encoder/ms-marco-TinyBERT-L-2-v2",
+    ):
+        try:
+            from sentence_transformers import CrossEncoder
+        except ImportError:
+            raise ImportError(
+                "Cannot import sentence-transformers package,",
+                "please `pip install sentence-transformers`",
+            )
+
+        self._top_n = top_n
+        self._model = CrossEncoder(model, max_length=512)
```

Comment:
I'm assuming we hard-coded this to 512 because all their models are 512 tokens?

Reply:
They honestly need to be clearer about this. It just happens to be what they recommend.